app/services/sso/jwt.rb

```diff
@@ -37,6 +37,7 @@ def bearer_token_set?
     end
 
     def no_issuer?
+      return true if jwt_token.decoded_payload.nil?
       !jwt_token.decoded_payload.key?('iss')
```

Comment:
Isn't this equivalent?
```suggestion
       !jwt_token.decoded_payload.try(:key?, 'iss')
```

Reply:
yeah, `try` seems to be a good idea!